config/configgrpc/configgrpc.go

```diff
@@ -39,6 +39,9 @@ type GRPCSettings struct {
 	// connection. See https://godoc.org/google.golang.org/grpc#WithInsecure.
 	UseSecure bool `mapstructure:"secure"`
 
+	// Authority to check against when doing TLS verification
+	ServerNameOverride string `mapstructure:"server-name-override"`
```

Comment:
@backjo it seems that you started before the recent merge of https://github.com/open-telemetry/opentelemetry-service/pull/357 we need to change this one to use underscore too. We should also get #359 to automatically validate this type of issue.

Reply:
Yep - will merge in latest and address